### PR TITLE
Feature/fix deprecation warnings

### PR DESCRIPTION
**Pull request by ekampf**

### CODE REVIEW

**Comment by ekampf** [comment]
FYI @syrusakbary

File: graphql/execution/tests/test_dataloader.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def batch_load_fn(self, keys):
     class Context(object):
         business_data_loader = BusinessDataLoader()
 
-    result = execute(schema, doc_ast, None, context_value=Context(), executor=executor)
+    result = execute(schema, doc_ast, None, context=Context(), executor=executor)
```
Comment (dan98765) [inline graphql/execution/tests/test_dataloader.py:75]: What's the purpose of removing `_value` from a bunch of the kwargs throughout this review? Is there some issue related to this where that's explained? A personal preference?

Comment (rhaven-pinterest) [reply graphql/execution/tests/test_dataloader.py:75]: Unless one has a foo and a foo_value, the suffix seems confusing. This is a Context instance, so the lower-case variable name seems a good fit

Comment (Cito) [reply graphql/execution/tests/test_dataloader.py:75]: The reason for this change is that these parameters have been renamed in graphql-core where the `_value` postfix was removed in June 2018 and you get deprecation warnings now if you use the `_value` postfix. That's why @ekampf removed the postfixes here as well.

However, I think we should revert the removal of `_value` in graphql-core - see #234.

**Comment by Cito** [comment]
Closing this in favor of #234.

**Pull request closed without merging**